content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,11 +9,24 @@ menu:
     parent: About the project
 ---
 
+## v1.8.5 [2020-04-19]
+
+The InfluxDB Enterprise 1.8.5 release builds on the InfluxDB OSS 1.8.5 release.
+For details on changes incorporated from the InfluxDB OSS release, see
+[InfluxDB OSS release notes](/influxdb/v1.8/about_the_project/releasenotes-changelog/#v185-2021-04-19).
+
+### Bug fixes
+
+- Resolve TSM backup "snapshot in progress" error.
+- Make meta queries respect QueryTimeout values.
```

Comment:
Is line 21 same as this (mentioned in OSS 1.8.5 release notes):

* Update meta queries (for example, SHOW TAG VALUES, SHOW TAG KEYS, SHOW SERIES CARDINALITY, SHOW MEASUREMENT CARDINALITY, and SHOW MEASUREMENTS) to check the query context when possible to respect timeout values set in the query-timeout configuration parameter. Note, meta queries will check the context less frequently than regular queries, which use iterators, because meta queries return data in batches.

Reply:
If yes, I believe we can drop here?

Reply:
It is the same, just making the new feature work in Enterprise, so feel free to drop here (or drop in OSS, whichever you prefer).

Reply:
Thank you, @davidby-influx, if added to both OSS 1.8.5 and Enterprise 1.8.5, will drop here.